Mathlib/Analysis/InnerProductSpace/Basic.lean

```diff
@@ -1572,6 +1572,26 @@ theorem toSesqForm_apply_norm_le {f : E →L[𝕜] E'} {v : E'} : ‖toSesqForm
 
 end ContinuousLinearMap
 
+variable (𝕜)
+
+/-- `innerSL` is an isometry. Note that the associated `LinearIsometry` is defined in
+`InnerProductSpace.Dual` as `toDualMap`. -/
+@[simp]
+theorem innerSL_apply_norm (x : E) : ‖innerSL 𝕜 x‖ = ‖x‖ := by
```

Comment:
Is this the only lemma that holds more generally for seminormed inner product spaces? I think a standalone PR that applies this generalization further would be a nice change, and could go in first.

Reply:
Right, when I introduced `PreInnerProduceSpace.Core`, I just changed `NormedAddCommGroup` to `SeminormedAddCommGroup` when the proof remained valid. Maybe I will look for more results that can hold with Semi with modified proofs.

Reply:
@eric-wieser I made #17007.